README.md

```diff
@@ -58,26 +34,7 @@ terraform plan # creates a plan consisting of a set of changes that will make yo
 terraform apply # performs a plan just like terraform plan does, but then actually carries out the planned changes to each resource using the relevant infrastructure provider's API
 ```
 
-## Developing the Provider
-
-If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (see [Requirements](#requirements) above).
-
-To compile the provider, see [Building The Provider](#building-the-provider).
-
-To add example docs, add the correspond `.tf` files to the `examples` directory. These should be added for every new data source and resource.
-
-To run terraform with the provider, create a `.terraformrc` file in your home directory (`~`) with the following content to override the provider installation with the local build:
-
-```hcl
-provider_installation {
-  dev_overrides {
-    "registry.terraform.io/astronomer/astro" = "~/terraform-provider-astro/bin" # Your path to the provider binary
-  }
-  direct {}
-}
-```
-
-## Example `main.tf` file for development and testing data sources and resources
+## Example `main.tf` file for testing data sources and resources
```

Comment:
if you're going to keep this section, you should include steps on how to test output with terraform plan and apply

otherwise this is useless and makes the section more confusing

like explain why this is here and what the point of this is

could also break down this section so it is more clear, like having one section for data source output, one section for resource - similar to how we have it in one of the guides (but then there is some overlap in content - so maybe it'd be better to point it to a guide? up to you)